### ISSUE TITLE
Mind Altar has rouge collision tile.

### ISSUE THREAD

**Issue by Zenpai93** [Skretzo/shortest-path#74]
There is a collision tile where there shouldn't be one causing unnecessary walking.  "who crafts mind runes?" well i am for right now lol.
I assume this is where i report this too.
![mindaltar](https://github.com/Skretzo/shortest-path/assets/169439926/3e832a0b-4af8-4eea-8697-9ee672614c85)

**Comment by Skretzo** [comment]
Yes, this is where you report issues like this. Looks like Jagex has actually mapped down an invisible wall there that is messing up my collision map. I might have to add an exception for that single tile.

**Comment by wvanderp** [comment]
see also https://github.com/Skretzo/shortest-path/issues/190

**Comment by FIrgolitsch** [comment]
Fixed in #466
